packages/extras/src/lib/components/GLTF/GLTF.svelte

```diff
@@ -30,7 +30,7 @@
   export let lookAt: GLTFProperties['lookAt'] = undefined
 
   export let url: GLTFProperties['url']
-  export let dracoDecoderPath: GLTFProperties['dracoDecoderPath'] = undefined
+  export let dracoDecoderPath: GLTFProperties['dracoDecoderPath'] = 'https://www.gstatic.com/draco/v1/decoders/';
```

Comment:
If you provide a default value for that prop, the loader which is instantiated in line 62 is always instantiated if the consumer does not provide `undefined` as a prop: `<GLTF dracoDecoderPath={undefined} />` which will lead to loading and instantiating of a DRACO decoder even if the user does not state that the model needs that.